models/src/main/protobuf/CasperMessage.proto

```diff
@@ -136,11 +136,17 @@ message Header {
   bytes extraBytes = 8;
 }
 
+message DeployCost {
+    bytes deployHash = 1; //Sha256 hash of the DeployString
```

Comment:
When I said tuple I meant the actual `Deploy` proto. This is fine too, but we have been using `Blake2b256` hashing for everything, so we had better stick with that here as well.

Reply:
I can change to `Blake2b256`. I didn't want to duplicate the data. If you prefer the actual `Deploy` that's no problem.